lms/djangoapps/ccx/views.py

```diff
@@ -94,8 +95,8 @@ def wrapper(request, course_id):
 
         # if there is a ccx, we must validate that it is the ccx for this coach
         if ccx is not None:
-            coach_ccx = get_ccx_for_coach(course, request.user)
-            if coach_ccx is None or coach_ccx.id != ccx.id:
+            coach_ccx = get_ccx_by_ccx_id(course, request.user, ccx.id)
+            if coach_ccx is None:
                 return HttpResponseForbidden(
```

Comment:
@macdiesel this part of code is not in scope of this PR, because it pre-exist. I havent modified/refactored it.

cc @pdpinch